docs/2.7.0/docs/canton/usermanual/troubleshooting_guide.rst

```diff
@@ -134,54 +134,39 @@ Log Files
 
      All Canton log statements contain a :ref:`trace-id <tracing>`. This tracing is turned on by default and the ``trace-id`` is passed between the distributed processes::
 
-          c.d.c.p.p.s.InFlightSubmissionTracker:participant=participant1 
-          tid:d5df95972a95b5ff00cb5cc3346c545f - NOT_SEQUENCED_TIMEOUT(2,d5df9597): 
-          Transaction was not sequenced within the pre-defined max sequencing time and has 
-          therefore timed out err-context:{location=SubmissionTrackingData.scala:175, 
-          timestamp=2022-10-19T17:45:56.393151Z} 
+          c.d.c.p.p.s.InFlightSubmissionTracker:participant=participant1
+          tid:d5df95972a95b5ff00cb5cc3346c545f - NOT_SEQUENCED_TIMEOUT(2,d5df9597):
+          Transaction was not sequenced within the pre-defined max sequencing time and has
+          therefore timed out err-context:{location=SubmissionTrackingData.scala:175,
+          timestamp=2022-10-19T17:45:56.393151Z}
 
-     In above example, we see the trace id twice: ``tid:d5df95972a95b5ff00cb5cc3346c545f`` and ``NOT_SEQUENCED_TIMEOUT(2,d5df9597)``. By filtering according to the ``trace-id``, you can find almost all log statements that relate to a particular command. Unfortunately, the ledger-api server does not yet log the ``trace-id``. Therefore, sometimes, we also need to find out the command id of a transaction. You can do that by grepping for the “rosetta stone”, which is one particular log line that contains both strings::
+     In above example, we see the trace id twice: ``tid:d5df95972a95b5ff00cb5cc3346c545f`` and ``NOT_SEQUENCED_TIMEOUT(2,d5df9597)``. By filtering according to the ``trace-id``, you can find almost all log statements that relate to a particular command. The ledger-api server logs the ``trace-id`` as well. However, sometimes, we also need to find out the command id of a transaction. You can do that by grepping for the “rosetta stone”, which is one particular log line that contains both strings::
 
-          2022-10-19 17:45:20,630 [⋮] DEBUG 
-          c.d.c.p.s.CantonSyncService:SimplestPingIntegrationTestInMemory/participant=participant1 
+          2022-10-19 17:45:20,630 [⋮] DEBUG
```

Comment:
This logline is not ideal to drive home the message. We should use ledger api INFO log line instead. For instance this
```
2023-07-04 12:03:26,517 [⋮] INFO  c.d.c.p.a.s.c.CommandSubmissionServiceImpl:participant=participant1 tid:35e389f0e41fd0273443dd866ff9e347 - Submitting commands for interpretation, commands -> {readAs: [], deduplicationPeriod: {duration: 'PT168H'}, submittedAt: '2023-07-04T10:03:26.514885Z', ledgerId: 'participant1', applicationId: 'CSsubmitAndWaitBasic', submissionId: 'CSsubmitAndWaitBasic-alpha-410b4d7b1b585-submission-0', actAs: ['CSsubmitAndWaitBasic-alpha-410b4d7b1b585-party-0::122035bd93d74879ce582adf5aa04a809b4b20618d39c1a9c2a17d35c29ab1ed098f'], commandId: 'CSsubmitAndWaitBasic-alpha-410b4d7b1b585-command-0', workflowId: 'CSsubmitAndWaitBasic-410b4d7b1b585'}.
```